demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -168,7 +168,7 @@ public void initiliazeProjectProductData() {
                     .project(project1)
                     .description("Test build config for project newcastle")
                     .environment(environment1)
-                    .buildScript("mvn clean deploy -Dmaven.test.skip")
+                    .buildScript("mvn clean deploy -DskipTests=true")
```

Comment:
Maybe but many community projects might be like this. We have to build what we are provided. Also we should probably deploy the same as them i.e. deploy the test artifacts.